cmd/edgectl/aes_install.go

```diff
@@ -80,6 +85,16 @@ func aesInstall(cmd *cobra.Command, args []string) error {
 	kcontext, _ := cmd.Flags().GetString("context")
 	i := NewInstaller(verbose)
 
+	// If Scout is disabled (environment variable set to non-null), inform the user.
+	// TODO: should be i.scout.Disabled() when Alex's PR #2496 is merged.
+	if os.Getenv("SCOUT_DISABLE") != "" {
+		i.show.Printf(color.Notice.Sprintf("[aesInstall] Scout is disabled; metrics will not be written to Metriton."))
+	}
+
+	// Both printed and logged when verbose.
+	i.log.Printf("[aesInstall] the install_id is: %s", i.scout.installID)
```

Comment:
Same comment here: the language should be user-centric. Something like: "[aesInstall] Info: unique id for this install: %s, %s". And it should only print this if Scout is enabled. And it should be just one line (one line is less "it must be important" to the user than two lines).

Reply:
Sounds good.